### DISCUSSION TITLE
Maui Icons

### DISCUSSION THREAD

**Discussion by siva546** [AathifMahir/MauiIcons#105] [General]
How can I add my own icons or reference icons from Resources\Images? When I provide image details in the value fields, the images are not displaying.

  <mi:MauiIcon
                x:Name="PART_Icon"
                HorizontalOptions="Center"
                IconAutoScaling="True"
                IconColor="#776C65"
                IconSize="24"
                Margin="0,10,0,0"
                VerticalOptions="Center">
                <mi:MauiIcon.Triggers>
               
                    <DataTrigger
                        Binding="{Binding Type, Source={x:Reference this}}"
                        TargetType="{x:Type mi:MauiIcon}"
                        Value="Home">
                        <Setter Property="Icon" Value="**OwnIcon.svg**" />
                    </DataTrigger>
                
                   
                </mi:MauiIcon.Triggers>
            </mi:MauiIcon>

**Comment by AathifMahir** [comment]
> How can I add my own icons or reference icons from Resources\Images? When I provide image details in the value fields, the images are not displaying.
> 
>   <mi:MauiIcon
>                 x:Name="PART_Icon"
>                 HorizontalOptions="Center"
>                 IconAutoScaling="True"
>                 IconColor="#776C65"
>                 IconSize="24"
>                 Margin="0,10,0,0"
>                 VerticalOptions="Center">
>                 <mi:MauiIcon.Triggers>
>                
>                     <DataTrigger
>                         Binding="{Binding Type, Source={x:Reference this}}"
>                         TargetType="{x:Type mi:MauiIcon}"
>                         Value="Home">
>                         <Setter Property="Icon" Value="**OwnIcon.svg**" />
>                     </DataTrigger>
>                 
>                    
>                 </mi:MauiIcon.Triggers>
>             </mi:MauiIcon>

Setters and Triggers were not supported yet by MauiIcons, We are currently on Research Stage on Implementating this feature. Of course this feature is Main Theme for v4 of MauiIcons.
https://github.com/AathifMahir/MauiIcons/issues/85
